addons/website/static/tests/tours/website_navbar_menu.js

```diff
@@ -0,0 +1,20 @@
+odoo.define("website.tour.website_navbar_menu", function (require) {
+"use strict";
+
+var tour = require("web_tour.tour");
+
+tour.register("website_navbar_menu", {
+    test: true,
+    url: "/",
+}, [
+    {
+        content: "Ensure menus are in DOM",
+        trigger: '#top_menu .nav-item a:contains("Test Tour Menu")',
+        run: function () {}, // it's a check
+    }, {
+        content: "Ensure menus loading is done (so they are actually visible)",
+        trigger: '#top_menu:not(.o_menu_loading)',
+        run: function () {}, // it's a check
+    }
```

Comment:
Maybe a unique check with trigger:
`#top_menu:not(.o_menu_loading) .nav-item:hasOpacity a:contains("Test Tour Menu") `

Little more robust (not that much but still). What do you think?

Reply:
They are split for a reason, they test different things, I'd not merge into one step (in which case I'd still separate them with extra_trigger tho).

About `hasOpacity`, I'd not add it, it is still as fragile as just using `o_menu_loading`, maybe a bit less but not a big diff, plus the feature could still be broken even if `hasOpacity` is true, if the height is still forced to 0.

I think this is okay as it is, it ensure the menu JS behavior ran as expected.

Reply:
Not as fragile, since if we move the o_menu_loading class elsewhere your test is still green, not with hasOpacity. But ok, better than no test :+1:

Reply:
EDIT: in fact why not just: `trigger: 'body:not(:has(.o_menu_loading))'` ? Still breakable by renaming the class, but not by moving it.